build.gradle

```diff
@@ -413,6 +412,7 @@ project(':iceberg-data') {
 
     testImplementation project(path: ':iceberg-api', configuration: 'testArtifacts')
     testImplementation project(path: ':iceberg-core', configuration: 'testArtifacts')
+    testImplementation libs.junit.vintage.engine
```

Comment:
`iceberg-data` still requires the dependency because code from this module is used in Flink tests